src/FbgemmSparseDenseVectorInt8Avx512.cc

```diff
@@ -205,14 +196,7 @@ void SparseDenseInt8MVAvx512(
         res = _mm512_add_epi32(res, c_i32_v);
       }
       // Horizontal reduce
-      // _mm512_reduce_add_epi32 is only available for gcc version > 7
-#if __GNUC__ >= 7
```

Comment:
Removing this ifdef turns out to break some internal downstream package builds, can we revert this change?

Reply:
Reverted